app/views/claims/_school_search.html.erb

```diff
@@ -45,7 +45,7 @@
   %>
 
   <%= form_for current_claim, url: claim_path do |form| %>
-    <%= form_group_tag current_claim, school_param do %>
+    <%= form_group_tag current_claim do %>
```

Comment:
I suspected this was the issue when I was refactoring. Glad to see my hunch was right :).